hermes-main-website/src/index.html

```diff
@@ -6,8 +6,8 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/png" href="assets/hermes_icon.png">
-  
-</head>
+  <link rel="stylesheet" href="styles.sass">
```

Comment:
styles.sass is already imported globally by the angular's engine so no need for this line, should be removed